.gitignore

```diff
@@ -2,10 +2,7 @@
 _antora
 
 ## Ignore commodore output
-catalog
```

Comment:
Can we leave them ignored? Otherwise all compiled changes and the inventory would be tracked by git.